packages/web-components/src/components/ic-button/ic-button.stories.mdx

```diff
@@ -1321,3 +1321,43 @@ export const defaultArgs = {
     `}
   </Story>
 </Canvas>
+
+### File Upload
+
+<Canvas>
+  <Story name="File Upload" parameters={{ loki: { skip: true } }}>
+    {html`
+      <script>
+        const fileBtn = document.querySelector("#file-button");
+        fileBtn.addEventListener("icFileSelection", (ev) => {
+          console.log(ev.detail);
+        });
+      </script>
+      <div style="padding: 6px">
+        <ic-button
```

Comment:
If possible, could this button emit an alert (using the file selection prop) containing the filename (or some other detail about the file) when a file is chosen? It would prove the functionality of the component :)